assimilation_code/programs/readme.rst

```diff
@@ -0,0 +1,195 @@
+Programs included with DART
+===========================
+
+This list of programs is separated into groups which have similar functionality.
+Within each group they are sorted  by the order
+in which they might be used andor by how widely they are used.
+
+Setting Up Experiments
+-----------------------------------
+
+In many cases, you won't need to use any programs in Setting Up Experiments
+except for ``preprocess``, because you're using an existing model interface 
+and have the observation sequence files.
+In that case, you the programs you're looking for are probably in `Assimilation Programs`_.
+ 
+`preprocess/preprocess <preprocess/preprocess.html>`_
```

Comment:
```suggestion
`preprocess <preprocess/preprocess.html>`_
```
for anything where the name of the executable and the subdirectory is the same, repeating the name just looks confusing.  for places where they differ, i'd keep the subdir name.